controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -1,3 +1,11 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
About it... I don't think it is right to squash the two commits because of license comment.
They are two different logic units and should be in separate commits.

Reply:
It was a neatpick anyway, so ok